CONTRIBUTING.md

```diff
@@ -28,6 +28,6 @@ If you work at Springer Nature, please remember to keep discussion inside the is
 
 ### Publicising your changes
 
-Changes to the playbook are relevant to all Front-End developers at Springer Nature, which is why once you've had work merged, the #frontend Slack room is notified, providing a link to the pull request.
+The #frontend Slack room is notified when work is merged, because changes to the playbook are relevant to all Front-End developers at Springer Nature.
```

Comment:
Using the active voice makes it explicit what or who is doing the acting upon something - the passive voice only describes what is being acted upon. 

Active voice: "**A bot/slack integration** notifies #frontend when **you** merge your work..."